compiler/src/iree/compiler/Dialect/HAL/Transforms/DumpExecutableBenchmarks.cpp

```diff
@@ -464,8 +470,13 @@ class DumpExecutableBenchmarksPass
     // filtering out dispatches that have dynamic parameters we don't
     // currently support.
     auto dispatchParamsMap = gatherDispatchParams(moduleOp);
-    if (dispatchParamsMap.empty())
+    if (dispatchParamsMap.empty()) {
+      llvm::dbgs()
+          << "Executable benchmarks were requested but none were generated. "
+             "Run with --debug-only=iree-dump-executable-benchmarks for more "
+             "details.\n";
```

Comment:
Also considered using `moduleOp.emitRemark()` here.

Reply:
llvm::dbgs is stripped in release builds so you'll want to use emitRemark, but not the module one as that prints the entire module out - `mlir::emitRemark(module.getLoc(), ...)` avoids printing all the IR.

Reply:
Ah, nice. Switched this place to use `emitRemark`. Still using `llvm::dbgs()` above.